appveyor.yml

```diff
@@ -2,26 +2,24 @@
 # subject to the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 #
-# Copyright Rene Rivera 2015-2016.
-
-build: false
+# Copyright Rene Rivera 2015-2017.
 
+ 
 image:
-  - Visual Studio 2013
-  - Visual Studio 2015
   - Visual Studio 2017
+  - Visual Studio 2015
+  - Visual Studio 2013
 
 environment:
   matrix:
-  - TOOLSET: msvc
-    TEST_ALL_EXTRAS:
+  - TEST_ALL_EXTRAS:
+
+build_script:
+- ps: |
+    & .\bootstrap.bat
```

Comment:
but you can check bootstrap.log in artifacts for that https://ci.appveyor.com/project/boostorg/build/build/1.0.94/job/ivfq7ou4nd5yu95n/artifacts

This is not enough?